pkg/acceptance/util.go

```diff
@@ -276,10 +276,27 @@ func MakeFarmer(t testing.TB, prefix string, stopper *stop.Stopper) *terrafarm.F
 		t.Fatalf("generated cluster name '%s' must match regex %s", name, prefixRE)
 	}
 
+	// We need to configure a MaxOffset on this clock so that the rpc.Context will
+	// enforce the offset. We're going to initialize client.Txn's using the
+	// Context's clock and send them through the ExternalSender, so the client's
+	// clock needs to be synchronized.
+	// TODO(andrei): It's unfortunate that this client, which is not part of the
+	// cluster, needs to do offset checks. Also, we igore the env variable that
+	// may control a different acceptable offset for the nodes in the cluster. We
+	// should stop creating transaction outside of the cluster.
+	clientClock := hlc.NewClock(hlc.UnixNano, base.DefaultMaxClockOffset)
 	rpcContext := rpc.NewContext(log.AmbientContext{}, &base.Config{
 		Insecure: true,
 		User:     security.NodeUser,
-	}, hlc.NewClock(hlc.UnixNano, 0), stopper)
+		// Set a bogus address, to be used by the clock skew checks as the ID of
+		// this "node". We can't leave it blank.
+		Addr: "127.0.0.1:4242",
+	}, clientClock, stopper)
+	rpcContext.HeartbeatCB = func() {
+		if err := rpcContext.RemoteClocks.VerifyClockOffset(context.Background()); err != nil {
```

Comment:
This seems suspicious as these are meant to measure a node against the mean of entire cluster and we'll be doing only a one on one comparison here. I think just setting max time stamp based on max offset should be sufficient.